From ef71208f8576a306c700c63afbee93f1669d74d2 Mon Sep 17 00:00:00 2001
From: Martin Sperl <kernel@martin.sperl.org>
Date: Tue, 26 Apr 2016 11:23:53 +0000
Subject: [PATCH 21/63] ARM: bcm2708: fix auxiliar clock provider to use clock
 framework

Fix the auxiliar clock provider to use the clock framework

Signed-off-by: Martin Sperl <kernel@martin.sperl.org>
---
 arch/arm/boot/dts/bcm2708_common.dtsi | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/bcm2708_common.dtsi b/arch/arm/boot/dts/bcm2708_common.dtsi
index ffdf782..e3decf8 100644
--- a/arch/arm/boot/dts/bcm2708_common.dtsi
+++ b/arch/arm/boot/dts/bcm2708_common.dtsi
@@ -231,12 +231,11 @@
 			status = "disabled";
 		};
 
-		aux: aux@0x7e215004 {
+		aux: aux@0x7e215000 {
 			compatible = "brcm,bcm2835-aux";
 			#clock-cells = <1>;
 			reg = <0x7e215000 0x8>;
-			clocks = <&clk_core>;
-			status = "disabled";
+			clocks = <&cprman BCM2835_CLOCK_VPU>;
 		};
 
 		uart1: serial@7e215040 {
-- 
2.7.4

